frontend/public/components/utils/index.tsx

```diff
@@ -37,3 +37,10 @@ export * from './deployment-pod-counts';
 export * from './entitlements';
 export * from './build-strategy';
 export * from './copy-to-clipboard';
+
+/* eslint-disable no-undef */
+export const enum NameValueEditorPair {
```

Comment:
This is used in other places as well. The tags modal also uses the enum.

Reply:
OK I see. I think I'd still leave it in name-value-editor.jsx as it was and just export it. I think it makes sense there because it's used specifically for that editor.

Reply:
Ok, will do.

Reply:
Spoke with Sam, keeping this here and adding a comment to document the reason its located here is to allow asynchronous loading of the editor